yb-voyager/src/srcdb/ora2pg_export_data.go

```diff
@@ -120,3 +120,56 @@ func extractAlterSequenceStatements(exportDir string) {
 
 	ioutil.WriteFile(filepath.Join(exportDir, "data", "postdata.sql"), []byte(requiredLines.String()), 0644)
 }
+
+// extract all identity column names from ALTER SEQUENCE IF EXISTS statements in postdata.sql
+func getIdentityColumnSequences(exportDir string) []string {
+	var identityColumns []string
+	alterSequenceRegex := regexp.MustCompile(`ALTER SEQUENCE (IF EXISTS )?(.*)? RESTART WITH`)
+
+	filePath := filepath.Join(exportDir, "data", "postdata.sql")
+	bytes, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		utils.ErrExit("unable to read file %q: %v\n", filePath, err)
+	}
+
+	lines := strings.Split(string(bytes), "\n")
+	for _, line := range lines {
+		if matches := alterSequenceRegex.FindStringSubmatch(line); matches != nil && strings.HasPrefix(matches[2], "iseq$$_") {
+			identityColumns = append(identityColumns, matches[2])
+		}
+	}
+
+	log.Infof("extracted identity columns list: %v", identityColumns)
+	return identityColumns
+}
+
+// replace all identity column names from ALTER SEQUENCE IF EXISTS statements in postdata.sql
+func replaceAllIdentityColumns(exportDir string, sourceTargetIdentitySequenceNames map[string]string) {
+	filePath := filepath.Join(exportDir, "data", "postdata.sql")
+	bytes, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		utils.ErrExit("unable to read file %q: %v\n", filePath, err)
+	}
+
+	lines := strings.Split(string(bytes), "\n")
+	for sourceSeq, targetSeq := range sourceTargetIdentitySequenceNames {
+		for i, line := range lines {
+			if strings.Contains(line, sourceSeq) {
+				lines[i] = strings.ReplaceAll(line, sourceSeq, targetSeq)
```

Comment:
Take this example line:
```
ALTER SEQUENCE IF EXISTS iseq$$_81022 RESTART WITH 11;
```
Is there any possibility, of another line having sequence number like `iseq$$_810220`:
```
ALTER SEQUENCE IF EXISTS iseq$$_810220 RESTART WITH 45;
```
In this case, the `strings.Contains(line, "iseq$$_81022")` will match to two lines.

Reply:
done